client/src/components/events/normal_events.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { Text, Tag, Spacer} from 'cornell-glue-ui'
+import LikeContainer from './like'
+import { FlexContainer } from 'cornell-glue-ui'
+import { IEvent, IEventDate } from 'src/types/event.type'
+import { IUser } from 'src/types/user.type'
+import './event.css'
+
+
+
+interface EventProps {
```

Comment:
as a convention, interfaces should have "I" as its first character. so this should be renamed IEventProps